### PR TITLE
Reinstate qTip Loading

### PR DESCRIPTION
**Pull request by tn3rb**
this PR:

- fixes the qTip loading which still needs to happen on the frontend for add-ons like the calendar that use it

### CODE REVIEW

**Review by knazart — DISMISSED** [review]
🏅

File: core/domain/services/assets/CoreAssetManager.php [hunk 1]
```diff
@@ -180,7 +179,13 @@ private function getCurrencySettings()
      * @param JavascriptAsset $script
      * @deprecated $VID:$
      */
-    public function loadQtipJs(JavascriptAsset $script)
+    public function loadQtipJs(JavascriptAsset $script = null)
     {
+        // qtip is turned OFF by default, but prior to the wp_enqueue_scripts hook,
+        // can be turned back on again via: add_filter('FHEE_load_qtip', '__return_true' );
+        if (apply_filters('FHEE_load_qtip', false)
+        ) {
```
Comment (knazart) [inline core/domain/services/assets/CoreAssetManager.php:187]:
```suggestion
        if (apply_filters('FHEE_load_qtip', false)) {
```

Can it be this ?

**Review by manzoorwanijk — APPROVED** [review]

File: core/domain/services/assets/CoreAssetManager.php (hunk 1 above)
Comment (tn3rb) [reply core/domain/services/assets/CoreAssetManager.php:187]: # 🤦🏻‍♂️ 

yes... yes it can

**Review by manzoorwanijk — APPROVED** [review]
🚢 it

**Pull request merged**